tests/scanner/audit/resource_rules_engine_test.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
Add a test for:
* Resource is present but under the wrong parent (e.g. project id is under the wrong folder/org)
* Expected child resource is missing.

Reply:
Done.